var/spack/repos/builtin/packages/openexr/package.py

```diff
@@ -87,6 +98,6 @@ def configure_args(self):
 
         return args
 
-    @when("@:2")
+    @when("@:2.3")
     def cmake(self, spec, prefix):
```

Comment:
This should really be a multi-build system package. `gdal` is my go-to example of this but there are many more.